Removed all hard-coded strings in .xml and .java files

Sommie09 opened the pull request:
### Description
All hardcoded strings were moved to the strings.xml file

Fixes #757  

### Type of Change:
**Delete irrelevant options.**
- Quality Assurance


**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
I ran the application using the Lint tool, there are no hardcoded strings available in the .java and .XML files


### Checklist:


- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] I have written Kotlin Docs whenever is applicable


**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

Sommie09 committed:
Removed all hard-coded strings in .xml and .java files

rpattath commented:
Fixes #757

rpattath commented:
@Sommie09 please format the following line correctly
```
Fixes # [757]
```
This will help to automatically close the issue and to indicate that you have opened a PR for this issue.

Sommie09 commented:
Done with the format @rpattath

Sommie09 commented:
Hello @rpattath could I go ahead with other issues?

Sommie09 committed:
docs: Created a Code_Of_Conduct.md file 

I have created a code of conduct.md file for Mentorship Android following the proper guidelines from the Systers Code of Conduct site, and also updated the link in README and CONTRIBUTING.md files

Sommie09 committed:
Delete CODE_OF_CONDUCT.md

sanchi0204 reviewed (changes requested):
Please change the branch name from develop to something more suitable to the issue mentioned.

Rest is good.

Sommie09 committed:
docs: Created a code_of_conduct.md file 

I have created a code of conduct.md file for Mentorship Android following the proper guidelines from the Systers Code of Conduct site.

Sommie09 committed:
Merge pull request #1 from Sommie09/add-code-of-conduct-1-1

docs: Created a code_of_conduct.md file

isabelcosta commented:
@Sommie09 can you remove the change you did to the code of conduct from this PR, so that it only contains the changes you are supposed to submit to solve issue #757 

About the branch name, usually, so you can contribute with multiple PRs, it's advised in our Contribution Guidelines that you create a separate branch for each contribution. So I would advise you if you want to contribute multiple times to this repository, that you create a new branch, apply the changes of this issue and submit a new PR, so that then you can have multiple parallel branches and not get into trouble with your git commit history when we squash and merge your contribution 🤗 

Let me know if anything is unclear.

cc @sanchi0204

sanchi0204 commented:
@Sommie09 pls update the PR with the changes suggested!

Sommie09 committed:
Delete CODE_OF_CONDUCT.md

Sommie09 commented:
@vj-codes I would love to close #922 as it is on another branch. Thank you

blundell commented on app/src/main/res/values/strings.xml:
```diff
@@ -211,5 +211,11 @@ We engage our community by contributing to open source, collaborating with the g
     <string name="bug">Bug</string>
     <string name="input"> Please fill in the input fields and/or assign a rating</string>
     <string name="feedback_thank">Thank you for your feedback!</string>
+    <string name="mentee_with_column">Mentee:</string>
```
I think this should be `colon` not `column` :-) https://en.wikipedia.org/wiki/Colon_(punctuation)

sanchi0204 commented:
> @vj-codes I would love to close #922 as it is on another branch. Thank you

@Sommie09 I suggest you to go with #922 as it follows the changes you were asked to do?
